docs/community/participate.md

```diff
@@ -0,0 +1,18 @@
+# GradleUp Community Channels
+
+## GitHub Issues
+
+Each project has GitHub issues.
+Please raise bugs and discuss feature requests there.
+
+## Public Channels
+
+!!! info "Coming Soon!"
+
+    Community Channels will be published soon, stay tuned.
+
+## Team Slack
+
+We have a private community Slack.
+If you are a maintainer or an active contributor,
+reach out to one of the maintainers to get added.
```

Comment:
Nit: I would remove this section. As an outside user, it makes me feel as if the community is gated (which it in fact isn't). Maintainers can already contact the team to get their projects added and we can add them at that time.

```suggestion
```